config.ini

```diff
@@ -17,7 +17,7 @@ webpush_channels.permission_backend = kinto.core.permission.memory
 [server:main]
 use = egg:waitress#main
 host = 127.0.0.1
-port = 9999
+port = 8888
```

Comment:
8888 is the port for kinto, we should use a different one for this service because both of them will be running together at some point.